### DISCUSSION TITLE
Are there any API for external control srsran stack?

### DISCUSSION THREAD

**Discussion by nikitos1550** [srsran/srsRAN_4G#1211] [Q&A]
Hi!

Are there some api or some other way to monitor some state of srsran?
I am planning using srsran based LTE test station (I already made some homework before and somehow already can setup software)
for testing my IoT devices. For tests is it interesting to get information about connected devices and possibly understand some metrics like RSSI and similar, for the network traffic stat it is clear that I can use ordinary Linux tools. As for control it is interesting to force device disconnect, allow/deny devices in realtime. 
For getting stat at least I can parse enable verbose logs and parse them in realtime, but for real time device management is it not clear for me, how can I do this. 
Maybe somebody, who is more familiar with srsran internals can gove some advices.

**Comment by natanzi** [comment]
I am curious as well.

**Comment by pemo75** [comment]
hi @nikitos1550
did you find any answer?
